services/bettersocial/fcmToken/sendNotification.js

```diff
@@ -22,11 +22,12 @@ const sendNotification = async (
       user_id: userIdFollowed,
       user_id_follower: userIdFollower,
       username_follower: userNameFollower
-    }
+    },
+    token: userTargetToken?.token
   };
   if (userTargetToken) {
     messaging()
-      .sendToDevice(userTargetToken?.token, payload)
+      .send(payload)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code uses the `send` method instead of `sendToDevice`. However, it's important to note that the `send` method requires a message object that includes the target (token, topic or condition), whereas `sendToDevice` directly takes the token as an argument. 

From the provided hunk, it seems like the `payload` object now includes a `token` property. This is correct for the `send` method, but there's no error handling if the `send` promise fails. It would be better to add a `.catch` block to handle any errors that might occur during the sending process.

```diff
   if (userTargetToken) {
     messaging()
      .send(payload)
-       .then(() => {});
+       .then(() => {})
+       .catch((error) => {
+         console.error('Error sending notification:', error);
+       });
   }
```
This way, you can log the error and take appropriate action if needed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->